parts/dcos/bstrap/bootstrapparams.t

```diff
@@ -12,22 +12,6 @@
       "type": "securestring"
       },
     {{end}}
-{{if IsHostedBootstrap}}
```

Comment:
I agree this change is pretty straight forward but I still think we should hold ourselves to not changing any legacy DCOS code without having an e2e test.

Reply:
I think this is just unacceptable cruft we should get rid of in order to streamline the code surface area.